### DISCUSSION TITLE
i get Unable to call lifecycle method [mount] directly on component

### DISCUSSION THREAD

**Discussion by mh24409** [livewire/livewire#2795] [1. Help]
Unable to call lifecycle method [mount] directly on component:[component name] how can i solve this error

**Comment by calebporzio** [comment]
You are restricted from calling those methods from the front end.

**Reply by mh24409** [reply]
I took the component from an old project that works well . but get that error 
how can i use this method or solve this error

**Reply by calebporzio** [reply]
This was considered a security hole. The patch was recently introduced

**Reply by mh24409** [reply]
can i solve this error now ?

**Reply by nuernbergerA** [reply]
You don't have too. Just put the logic you call in mount in another public function.

Call it from mount and the ui done.

**Reply by ArtMin96** [reply]
You can't call wire:poll="mount" or wire:poll="render".

Hook mount: Runs once, immediately after the component is instantiated, but before render() is called.
Check these two options.
